src/gax-internal/src/api_header.rs

```diff
@@ -48,7 +48,25 @@ impl XGoogApiClient {
         let gax_version = build_info::PKG_VERSION;
 
         format!(
-            "gl-rust/{rustc_version} gax/{gax_version} {}/{}",
+            "gl-rust/{rustc_version} gax/{gax_version} rest/{gax_version}-reqwest {}/{}",
+            self.library_type, self.version
+        )
+    }
+
+    /// Format the struct as needed for the `x-goog-api-client` header.
+    pub fn grpc_header_value(&self) -> String {
+        // Strip out the initial "rustc " string from `RUSTC_VERSION`. If not
+        // found, leave RUSTC_VERSION unchanged.
+        let rustc_version = build_info::RUSTC_VERSION;
+        let rustc_version = rustc_version
+            .strip_prefix("rustc ")
+            .unwrap_or(build_info::RUSTC_VERSION);
+
+        // Capture the gax version too.
+        let gax_version = build_info::PKG_VERSION;
+
+        format!(
+            "gl-rust/{rustc_version} gax/{gax_version} grpc/{gax_version}-tonic {}/{}",
```

Comment:
Same Q here about `tonic`

Reply:
Same answer.  We can use either, and I cannot find a way to capture the tonic version.